go-controller/hybrid-overlay/pkg/controller/node.go

```diff
@@ -90,12 +90,13 @@ func NewNode(
 	nodeInformer cache.SharedIndexInformer,
 	localPodInformer cache.SharedIndexInformer,
 	eventHandlerCreateFunction informer.EventHandlerCreateFunction,
+	isHONode bool,
```

Comment:
Can we add godoc text for this parameter or even enhance a bit for the whole thing as its the exported method of the package after all? Something like the following suggestion or whatever you think its best. I just think we are missing an explanation here.
```
// NewNode returns a new node controller
// This controller is designed to be used both by ovnkube-node binary and by the HO binary.
// When used by ovnkube-node binary, it prepares the OVN nodes for the HO tunnel.
// When used by the HO binary, it prepares the windows or SDN (SDN <-> OVN migration) nodes for the HO tunnel. This is flag by setting isHONode to true

// TODO(jtanenba) the localPodInformer no longer selects only local pods
```

Reply:
fixed